pyccel/ast/headers.py

```diff
@@ -14,6 +14,8 @@
 from .datatypes import datatype, DataTypeFactory, UnionType
 from .macros import Macro, MacroShape, construct_macro
 from .core import local_sympify
+from pyccel.errors.errors import Errors
```

Comment:
Codacy wants us to place absolute imports before relative imports. I think that makes sense, and it's probably recommended by some Python PEP out there. Could you please reorder them?

Reply:
You can also change this line to `from ..errors.errors import Errors` if you prefer, but I suspect it will still want it reordered